content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-cordform.md

```diff
@@ -84,11 +84,41 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
 ```
 
 {{< note >}}
-Make sure to use Corda gradle plugin version 5.0.10 or above.
+Make sure to use Corda Gradle plugin version 5.0.10 or above.
 {{< /note >}}
 
 The configuration values used in the example are described below.
 
+You can also turn this behaviour off by adding the following code to your node configuration:
```

Comment:
I cannot see what "this" refers to here. It sounds like were talking about deploying the local project's CorDapp (should one exist).

Reply:
made clear